### PR TITLE
Feature/transfers count

### LINKED ISSUE #8: Add transfers count to GetTransfersInfoResponse

### PR DESCRIPTION
**Pull request by yoavrozner**

### CODE REVIEW

File: src/transfer/transfer.repository.ts [hunk 1]
```diff
@@ -11,38 +11,71 @@ export class TransferRepository {
...
 
+  static async getSize(filter: any): Promise<number> {
+    const aggregationQuery : any[] = [
+      {
+        $match: filter
+      },
+      {
+        $group: {
+          _id: '$reqID',
+          docs:{
```
Comment (nettashviro) [inline src/transfer/transfer.repository.ts]: there is no need to specify the "docs", you only want to return the amount of documents

File: src/dropbox/dropbox.methods.ts [hunk 2]
```diff
@@ -160,16 +162,16 @@ export class DropboxMethods {
...
-      destUsers.map(async (destUser: IApprovalUser) => {
-        try {
-          await getUser(destUser.id, destination);
-        } catch (error) {
-          throw new ClientError(`cant get user: ${destUser.id} with destination: ${destination}`);
-        }
-      })
-    );
+    // if (destUsers.length < 1) throw new ClientError('must require at least 1 dest user');
+    // await Promise.all(
```
Comment (nettashviro) [inline src/dropbox/dropbox.methods.ts]: uncomment

File: src/dropbox/dropbox.methods.ts [hunk 3]
```diff
@@ -191,17 +193,17 @@ export class DropboxMethods {
...
-      to: params.users,
-      from: params.sharerID,
-      info: params.info,
-      classification: params.classification,
-    };
-    await approvalService.createRequest(request, destination);
+    // const request: IRequest = {
+    //   approvers,
+    //   id: reqID,
+    //   fileId: params.fileID,
```
Comment (nettashviro) [inline src/dropbox/dropbox.methods.ts]: uncomment

**Review by nettashviro — CHANGES_REQUESTED** [review]

**Review by Shahar-Y — APPROVED** [review]

**Comment by Shahar-Y** [comment]
@yoavrozner great job!

**Pull request merged**